gradebook/grade/models.py

```diff
@@ -0,0 +1,112 @@
+from django.db import models
+
+class User(auth.models.AbstractUser):
+    id = models.IntegerField()
+    email = models.CharField(max_length=200)
+    username = models.CharField(max_length=100)
+    first_name = models.CharField(max_length=100)
+    last_name = models.CharField(max_length=100)
+    password = models.CharField(max_length=100)
+    user_type = models.CharField(max_length=50)
+
+    class Meta:
+        # Prevents creation of table in db
+        # Removed when sub classed
+        # https://docs.djangoproject.com/en/3.2/topics/db/models/#abstract-base-classes
+        abstract = True
+
+class GradeSystem(models.Model):
+    pass
+
+class Teacher(models.Model):
+    pass
+
+
+class Student(models.Model):
+    pass
+
+class Subject(models.Model):
+    pass
```

Comment:
These should inherit from `User` to make them one-to-one relation.

If they don't contain any data right now, we should make them a proxy.